### ISSUE TITLE
At 200% zoom data present in the "Details view" is truncating and not visible clearly.: A11y_.NetAspireDashboard_Resources_Detailsview_Zoom

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/aspire#3395]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**

### Tags:
#A11yTCS; #A11yMAS; #A11ySev2; #FTP; #Win11; #ChromiumEdge; #FTP; #DesktopWeb; #.NET Core; #SH_.Net Aspire Dashboard_Web_Mar2024; #WCAG1.4.4; #Zoom;

### Prerequisites:
Display resolution: 1280* 768.

### Environment Details:
Application Name: .Net Aspire Dashboard
URL: https://localhost:19888/
Chrome Version 122.0.2365.80 (Official build) (64-bit)
Window Version: Win 11
OS Build: 22621.3085

### Repro Steps:

1. Hit the URL: http://localhost:15888/.
2. TAB to "Resources" tab item and hit select the tab by pressing Enter key.
3. TAB to "View" button present under 'Details' column header in the page and hit ENTER key.
4. "Details container" will open.
5. Zoom the page to 200% and observe the issue.

### Actual Results:
At 200% zoom data present in the "Details view" is truncating and not visible clearly.

### Expected Results:
At 200% zoom data present in the "Details view" should not be truncating and should be visible clearly.

### User Impact:
Low vision users will not be able to see the data present in the section clearly and will miss the information about it.

### Attachment:
[3395_A11y_.NetAspireDashboard_Resources_Detailsview_AfterZoom.webm](https://github.com/dotnet/aspire/assets/90901519/7c44d22c-7e84-4b85-b141-9ded590f1c12)

![3395_A11y_ NetAspireDashboard_Resources_Detailsview_BeforeResize](https://github.com/dotnet/aspire/assets/90901519/adfdcf6c-b162-489b-9a15-8bf869e2250f)

**Comment by AnKushSingh05** [comment]
#A11YCAIOH; #CEACCReviewed;

**Comment by adamint** [comment]
This has been fixed in `main`. Closing, @AnKushSingh05 please review

**Comment by merriemcgaw** [comment]
@PoojaNamde can you please verify this issue and tag it as closed?

**Comment by PoojaNamde** [comment]
@adamint Verified this issue in the latest environment: https://localhost:15887/. Here issue is not fixed and some of information is truncating like container image and container argument are truncating. Please find the below attachment for reference:

[Video 9-06 at 22.15.webm](https://github.com/user-attachments/assets/ef995d13-5742-464e-b674-20b4759c1b7b)

**Comment by adamint** [comment]
> @adamint Verified this issue in the latest environment: https://localhost:15887/. Here issue is not fixed and some of information is truncating like container image and container argument are truncating. Please find the below attachment for reference:
> 
>  Video.9-06.at.22.15.webm

I do not believe this is correct, since 
- in the video, the column widths were not modified, and 
- there is an accessible way to view the content (the text visualizer)
<img width="1017" alt="image" src="https://github.com/user-attachments/assets/b66b5fc1-ff7f-4089-8a28-2c12e9534190">
